tgui/packages/tgui/interfaces/Manufacturer/index.tsx

```diff
@@ -0,0 +1,321 @@
+                      <BlueprintButton
+                        key={index}
+                        blueprintData={blueprint}
+                        manufacturerSpeed={data.speed}
+                        materialData={data.resource_data}
+                      />
+                    ))}
+                  </Collapsible>
+                )
+              ))}
+            </Section>
+          </Stack.Item>
+          <Stack.Item grow>
+            <Stack vertical>
+              <Stack.Item>
+                <Input placeholder="Search..." width="100%" onInput={(_, value) => setSearchData(value)} />
+              </Stack.Item>
+              <Stack.Item>
+                <Section title="Loaded Materials" textAlign="center">
+                  <LabeledList>
+                    {data.resource_data.map((resourceData: Resource) => (
+                      <LabeledList.Item
+                        key={resourceData.id}
+                        buttons={[
+                          <Button
+                            key="eject"
+                            icon="eject"
+                            onClick={() => act("material_eject", { "resource": resourceData.id })}
+                          />,
+                          <Button
+                            key="swap"
+                            icon="arrows-up-down"
+                            color={(swap !== resourceData.id) ? null : "green"}
+                            onClick={() => swapPriority(resourceData.id)}
+                          />,
+                        ]}
+                        label={toTitleCase(resourceData.name)}
+                        textAlign="center"
+                      >
+                        {resourceData.amount}
+                      </LabeledList.Item>
+                    ))}
+                  </LabeledList>
+                </Section>
+              </Stack.Item>
+              <Stack.Item>
+                <Section
+                  textAlign="center"
+                  title={"Fabricator Settings"}
+                >
+                  <LabeledList>
+                    <LabeledList.Item
+                      label={"Repeat"}
+                      buttons={<Button icon="repeat" onClick={() => toggleRepeat()}>Toggle Repeat</Button>}
+                      textAlign="center"
+                    >
+                      {data.repeat ? "On" : "Off"}
+                    </LabeledList.Item>
+                    <LabeledList.Item
+                      label="Speed"
+                    >
+                      <Slider
+                        minValue={1}
+                        value={data.speed}
+                        maxValue={data.hacked ? data.max_speed_hacked : data.max_speed_normal}
+                        step={1}
+                        stepPixelSize={100}
+                        disabled={data.mode !== "working"}
+                        onChange={(_e: any, value: number) => act("speed", { "value": value })}
```

Comment:
```suggestion
                        onChange={(_e: any, value: number) => act("speed", { value })}
```
Shorthand for where the object's key and the value variable name are the same.